packages/account/src/test-utils/test-coin.ts

```diff
@@ -0,0 +1,51 @@
+import { Coin } from '@fuel-ts/account';
+import { bn, type BN } from '@fuel-ts/math';
+import { getRandomB256 } from '@fuel-ts/crypto';
+
+interface TestCoinSpecs {
+  id: string;
+  owner: string;
+  amount: BN;
+  type: number;
+}
+
+export class TestCoin {
+  public readonly id: string;
+  public readonly owner: string;
+  public readonly amount: BN;
+  public readonly type: number;
+

```

Comment:
The `TestCoin` should follow the same approach as the existent `TestMessage`:

https://github.com/FuelLabs/fuels-ts/blob/01375ce29bf5f18b5d8f5e97fec5d82f4fb6ae5f/packages/account/src/test-utils/test-message.ts#L18-L24

It is intended to be used when setting up a local node, just like the `TestMessage` is used:

https://github.com/FuelLabs/fuels-ts/blob/01375ce29bf5f18b5d8f5e97fec5d82f4fb6ae5f/apps/docs/src/guide/provider/snippets/functionality/get-messages-by-nonce.ts#L4-L17

Therefore, its properties should be the same ones as the [Coin type for the stateConfig](https://github.com/FuelLabs/fuel-core/blob/05538a5de35f8b9ec5d46f9fc37b043b1c6a566e/version-compatibility/forkless-upgrade/chain-configurations/v36/state_config.json#L4-L10):